### ISSUE TITLE
Uncaught TypeError: Cannot read property 'id' of undefined

### ISSUE THREAD

**Issue by ghost** [callahanrts/slack-chat#18]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.211.0
**System**: Ubuntu 15.04
**Thrown From**: [slack-chat](https://github.com/callahanrts/slack-chat) package, v3.2.0
### Stack Trace

Uncaught TypeError: Cannot read property 'id' of undefined

```
At /home/peduxe/.atom/packages/slack-chat/lib/views/conversations/member-view.coffee:7

TypeError: Cannot read property 'id' of undefined
  at Function.module.exports.MemberView.content (/home/peduxe/.atom/packages/slack-chat/lib/views/conversations/member-view.coffee:7:28)
  at Function.<anonymous> (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:156:31)
  at Function.View.buildHtml (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:123:10)
  at MemberView.View (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:155:33)
  at new MemberView (/home/peduxe/.atom/packages/slack-chat/lib/views/conversations/member-view.coffee:6:3)
  at ConversationView.module.exports.ConversationView.getMembers (/home/peduxe/.atom/packages/slack-chat/lib/views/conversation-view.coffee:37:29)
  at ConversationView.getMembers (/home/peduxe/.atom/packages/slack-chat/lib/views/conversation-view.coffee:2:1)
  at ConversationView.module.exports.ConversationView.initialize (/home/peduxe/.atom/packages/slack-chat/lib/views/conversation-view.coffee:20:6)
  at ConversationView.View (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at ConversationView.ScrollView (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/scroll-view.js:12:47)
  at new ConversationView (/home/peduxe/.atom/packages/slack-chat/lib/views/conversation-view.coffee:8:3)
  at StateController.module.exports.StateController.stateDefault (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:168:26)
  at StateController.stateDefault (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:2:1)
  at StateController.module.exports.StateController.setState (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:134:24)
  at StateController.setState (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:2:1)
  at StateController.module.exports.StateController.hello (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:69:6)
  at StateController.hello (/home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:2:1)
  at /home/peduxe/.atom/packages/slack-chat/lib/state-controller.coffee:56:7
  at WebSocket.<anonymous> (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/lib/app.js:132:27)
  at emitTwo (events.js:87:13)
  at WebSocket.emit (events.js:169:7)
  at Receiver.ontext (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/WebSocket.js:797:10)
  at /home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:473:18
  at Receiver.applyExtensions (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:360:5)
  at /home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:462:14
  at Receiver.flush (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:336:3)
  at Receiver.opcodes.1.finish (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:478:12)
  at Receiver.expectHandler (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:453:31)
  at Receiver.add (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/Receiver.js:91:24)
  at TLSSocket.firstHandler (/home/peduxe/.atom/packages/slack-chat/node_modules/sc-client/node_modules/ws/lib/WebSocket.js:774:22)
  at emitOne (events.js:77:13)
  at TLSSocket.emit (events.js:166:7)
  at readableAddChunk (_stream_readable.js:146:16)
  at TLSSocket.Readable.push (_stream_readable.js:109:10)
  at TLSWrap.onread (net.js:517:20)

```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "script",
      "polymer-snippets",
      "tool-bar-main",
      "filetype-color",
      "pigments"
    ],
    "themes": [
      "one-dark-ui",
      "atom-material-syntax"
    ]
  },
}
```
### Installed Packages

``` coffee
# User
.bin, vundefined
Sublime-Style-Column-Selection, v1.3.0
atom-material-syntax, v0.1.1
atom-ternjs, v0.6.4
atom-typescript, v4.6.2
autoclose-html, v0.18.0
autocomplete-paths, v1.0.2
color-picker, v2.0.7
emmet, v2.3.10
file-icons, v1.5.8
highlight-selected, v0.10.1
linter, v1.0.1
linter-csslint, v0.0.13
linter-htmlhint, v0.0.16
linter-jshint, v0.1.6
linter-php, v0.0.15
slack-chat, v3.2.0
tabs-to-spaces, v0.11.0
terminal-panel, v1.14.1

# Dev
No dev packages
```

**Comment by ghost** [comment]
Got this exception after authorizing the app.

**Comment by callahanrts** [comment]
Should be resolved in [v3.2.1](https://github.com/callahanrts/slack-chat/tree/v3.2.1). If you're missing any team members in slack-chat, post back with more information about them. It looks like this error is caused from a team member that doesn't have a channel id anymore. Banned or deleted members maybe?
